src/filters/color/ColorMatrixFilter.js

```diff
@@ -156,18 +150,19 @@ ColorMatrixFilter.prototype.brightness = function(b, multiply)
  * @param scale {number} value of the grey (0 is black)
  * @param multiply {boolean} refer to ._loadMatrix() method
  */
-ColorMatrixFilter.prototype.greyscale = function(scale, multiply)
+ColorMatrixFilter.prototype.greyscale = function (scale, multiply)
 {
     var matrix = [
         scale, scale, scale, 0, 0,
         scale, scale, scale, 0, 0,
         scale, scale, scale, 0, 0,
-        0,0,0,1,0,
-        0,0,0,0,1
+        0, 0, 0, 1, 0
     ];
 
     this._loadMatrix(matrix, multiply);
 };
+//Americanized alias
+ColorMatrixFilter.prototype.grayscale = ColorMatrixFilter.prototype.greyscale;
```

Comment:
I was half thinking I should now go through and make colour aliases...maybe a little extreme though